docs/02-app/01-building-your-application/07-configuring/01-typescript.mdx

```diff
@@ -78,15 +96,16 @@ Next.js can statically type links to prevent typos and other errors when using `
 
 To opt-into this feature, `experimental.typedRoutes` need to be enabled and the project needs to be using TypeScript.
 
-```js filename="next.config.js"
-/** @type {import('next').NextConfig} */
-const nextConfig = {
+```ts filename="next.config.ts"
```

Comment:
Didn't we have an affordance in docs to display the same file with two languages? We should show that you can still use typed JS via JSDoc.

Reply:
I think it's a good point.